cmd/maya-apiserver/spc-watcher/handler.go

```diff
@@ -139,3 +147,40 @@ func (c *Controller) getSpcResource(key string) (*apis.StoragePoolClaim, error)
 	}
 	return spcGot, nil
 }
+
+func syncSpc(spcGot *apis.StoragePoolClaim) error {
+	if len(spcGot.Spec.Disks.DiskList) > 0 {
+		// TODO : reconciliation for manual storagepool provisioning
+		return fmt.Errorf("no reconciliation for manual provisioning for storagepoolclaim %s", spcGot.Name)
```

Comment:
Since this is not really an error, will returning this result in the object being queued continuously?

Reply:
fixed